src/app/zosmf-server-config/zosmf-server-config.component.ts

```diff
@@ -78,6 +80,21 @@ export class ZosmfServerConfigComponent {
     this.configService.saveConfig(this.config);
   }
 
+  unsavedChanges(): boolean {
+    if (this.configService.getLocalConfig().zosmfServers.length != this.config.zosmfServers.length)
+      { return true; }
+    if (this.configService.getLocalConfig().defaultZosmfServer.host != this.config.defaultZosmfServer.host &&
+        this.configService.getLocalConfig().defaultZosmfServer.port != this.config.defaultZosmfServer.port)
+      { return true; }
+    for (let check = this.config.zosmfServers.length; check--;)
+    {
+      if (this.config.zosmfServers[check].host != this.configService.getLocalConfig().zosmfServers[check].host &&
```

Comment:
The same as above: AND(&&) must be changed to OR(||)

Reply:
Right. Silly mistake, thank you.